charts/observability-pipelines-worker/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+# 1.2.0
+
+* Update `run` subcommand to use new `--pipeline-config` argument
+* Rename `config` to `pipelineConfig` in values
+* Add `datadog.pipeline_id` value to replace `datadog.config_key`
```

Comment:
```suggestion
* Add `datadog.pipelineId` value to replace `datadog.configKey`
```

Or with the current styling of `pipelineID`, if you still prefer that. Probably also worth noting in the change log that `configKey` should still works for backwards compatibility?

Reply:
Sounds good, adjusted in [7a5e07c](https://github.com/DataDog/helm-charts/pull/952/commits/7a5e07c5b852e9393a9f49a75395fe08597c52dd)